--- tiki-index.php	2010-03-18 11:48:32.000000000 -0300
+++ tiki-index.php	2010-04-10 20:33:36.000000000 -0300
@@ -456,6 +456,19 @@
 		$pageRenderer->setShowAttachments( $_REQUEST['atts_show'] );
 }
 
+// ESTUDIOLIVRE HACK BEGIN - associa categoria ao smarty, pra fazer theme control na mao
+$tc_categs = $categlib->get_object_categories($cat_type, $cat_objid);
+
+if (count($tc_categs)) {
+    foreach ($tc_categs as $cat) {
+	$catt=$categlib->get_category($cat);
+	$smarty->assign_by_ref('category', $catt["name"]);
+	break;
+    }
+}
+// ESTUDIOLIVRE HACK END
+
+
 // Watches
 if ($prefs['feature_user_watches'] == 'y') {
 	if($user && isset($_REQUEST['watch_event']) && !isset($_REQUEST['watch_group'])) {
