--- comments.php	2010-03-18 11:48:32.000000000 -0300
+++ comments.php	2010-04-10 20:44:05.000000000 -0300
@@ -301,8 +301,11 @@
 	}
 }
 
+if ($_REQUEST["comments_threadId"] > 0)
+    $comment_info = $commentslib->get_comment($_REQUEST["comments_threadId"]);
+
 if (($tiki_p_remove_comments == 'y' && (!isset($forum_mode) || $forum_mode == 'n'))
-		|| (isset($forum_mode) && $forum_mode =='y' && $tiki_p_admin_forum == 'y' ) ) {
+		|| (isset($forum_mode) && $forum_mode =='y' && $tiki_p_admin_forum == 'y' ) || (isset($comment_info) && $user == $comment_info["userName"])) {
 	if (isset($_REQUEST["comments_remove"]) && isset($_REQUEST["comments_threadId"])) {
 		$area = 'delcomment';
 		if ($prefs['feature_ticketlib2'] != 'y' or (isset($_POST['daconfirm']) and isset($_SESSION["ticket_$area"]))) {
