--- lib/freetag/freetaglib.php.orig	2010-03-01 17:32:05.000000000 -0300
+++ lib/freetag/freetaglib.php	2010-06-07 14:43:19.000000000 -0300
@@ -959,13 +959,14 @@
 	{
 
 		// get top tag popularity
-		$query = 'SELECT COUNT(*) as count'
-						. ' FROM `tiki_freetagged_objects` o'
-						. ' GROUP BY `tagId`'
-						. ' ORDER BY count DESC'
-						;
-
-		$top = $this->getOne($query);
+		 $query = "SELECT COUNT(*) as count
+                        FROM `tiki_freetagged_objects` o
+                        WHERE 1=1 $mid
+                        GROUP BY tagId
+                        ORDER BY count DESC
+                        ";
+		
+		$top = $this->getOne($query, $bindvals);
 
 		$bindvals = array();
 
@@ -1510,4 +1511,8 @@
 	}
 }
 
-$freetaglib = new FreetagLib;
+require_once("lib/freetag/eltaglib.php");
+
+global $dbTiki;
+
+$freetaglib = new ElTagLib($dbTiki);
