--- lib/tikilib.php	2006-06-04 22:28:58.000000000 -0300
+++ lib/tikilib.php	2006-09-19 18:37:07.000000000 -0300
@@ -2115,8 +2115,8 @@
 
 	// Fixed the not null. After some research i came to the conclusion that only these dbms that i use
 	// lack a ansi outer join syntax. I guess using outer joins should be ok. redflo.
-	if (substr($sort_mode,1,7)=="country") {
-		$query = "select u.*, f.`user` as friend, p.`value` as country from `users_users` as u left join `tiki_friends` as f on u.`login`=f.`friend` and f.`user`=? left join `tiki_user_preferences` p on u.`login`=p.`user` and p.`prefName`='country' $mid order by $sort_mode";
+	if (substr($sort_mode,1,5)=="local") {
+		$query = "select u.*, f.`user` as friend, p.`value` as local from `users_users` as u left join `tiki_friends` as f on u.`login`=f.`friend` and f.`user`=? left join `tiki_user_preferences` p on u.`login`=p.`user` and p.`prefName`='local' $mid order by $sort_mode";
 	} else {
 		$query = "select u.*, f.`user` as friend, p.`value` as realName from `users_users` as u left join `tiki_friends` as f on u.`login`=f.`friend` and f.`user`=? left join `tiki_user_preferences` p on u.`login`=p.`user` and p.`prefName`='realName' $mid order by $sort_mode";
 	}
--- tiki-list_users.php.old	2011-04-04 10:52:20.328370993 -0300
+++ tiki-list_users.php	2011-04-04 10:52:57.048370996 -0300
@@ -54,10 +54,10 @@
 	}
 	if ($prefs['feature_community_list_country'] == "y") {
 		$userprefs = $listusers["data"][$i]["preferences"];
-		$country = "None";
+		$country = "";
 		
 		for ($j = 0, $count_userprefs = count($userprefs); $j < $count_userprefs; $j++) {
-			if ($userprefs[$j]["prefName"] == "country") $country = $userprefs[$j]["value"];
+			if ($userprefs[$j]["prefName"] == "local") $country = $userprefs[$j]["value"];
 			if ($userprefs[$j]["prefName"] == "realName") $listusers["data"][$i]["realName"] = $userprefs[$j]["value"];
 		}
 	}
