--- tiki-user_preferences.php	2010-03-18 16:55:12.000000000 -0300
+++ tiki-user_preferences.php	2010-06-07 15:17:19.000000000 -0300
@@ -123,6 +123,13 @@
 		$tikilib->set_user_preference($userwatch, 'user_dbl', 'n');
 		$smarty->assign('user_dbl', 'n');
 	}
+	if (isset($_REQUEST['user_useEditJs']) && $_REQUEST['user_useEditJs'] == 'on') {
+               $tikilib->set_user_preference($userwatch, 'user_useEditJs', 'y');
+               $smarty->assign('user_useEditJs', 'y');
+        } else {
+               $tikilib->set_user_preference($userwatch, 'user_useEditJs', 'n');
+               $smarty->assign('user_useEditJs', 'n');
+        }
 	if (isset($_REQUEST['diff_versions']) && $_REQUEST['diff_versions'] == 'on') {
 		$tikilib->set_user_preference($userwatch, 'diff_versions', 'y');
 		$smarty->assign('diff_versions', 'y');
@@ -338,6 +345,8 @@
 $tikilib->get_user_preference($userwatch, 'user_dbl', 'y');
 $userinfo = $userlib->get_user_info($userwatch);
 $smarty->assign_by_ref('userinfo', $userinfo);
+$user_useEditJs = $tikilib->get_user_preference($userwatch, 'user_useEditJs', 'n');
+$smarty->assign_by_ref('user_useEditJs', $user_useEditJs);
 $llist = array();
 $llist = $tikilib->list_styles();
 $smarty->assign_by_ref('styles', $llist);
