--- lib/smarty_tiki/modifier.avatarize.php.orig	2010-02-14 16:16:23.000000000 -0200
+++ lib/smarty_tiki/modifier.avatarize.php	2010-06-07 17:48:11.000000000 -0300
@@ -26,7 +26,7 @@
   $avatar = $tikilib->get_user_avatar($user);
   if ( $avatar != '' && $tikilib->get_user_preference($user, 'user_information', 'public') == 'public' ) {
 	$id = $userlib->get_user_id($user);
-  	$avatar = "<a title=\"$user\" href=\"tiki-user_information.php?userId=$id\">".$avatar.'</a>';
+  	$avatar = "<a title=\"$user\" href=\"el-user.php?view_user=$user\">".$avatar.'</a>';
   } 
   return $avatar;	
 }
