--- ../estudiolivre_base/tiki-editpage.php	2006-06-07 18:57:52.000000000 -0300
+++ tiki-editpage.php	2006-07-25 17:48:02.000000000 -0300
@@ -872,8 +874,15 @@
 	$smarty->assign('anon_user', 'y');
 }
 
+if (isset($_REQUEST["freetag_string"])) {
+  $smarty->assign_by_ref('taglist', $_REQUEST["freetag_string"]);
+}
+
 ask_ticket('edit-page');
 
+$useJs=$tikilib->get_user_preference($user,'user_useEditJs');
+$smarty->assign('useJavascript',$useJs);
+
 // Display the Index Template
 $smarty->assign('mid', 'tiki-editpage.tpl');
 $smarty->assign('show_page_bar', 'y');
