--- tiki-searchresults.php	2010-03-16 23:23:40.000000000 -0300
+++ tiki-searchresults.php	2010-06-07 20:33:36.000000000 -0300
@@ -214,6 +214,7 @@
 // disallow robots to index page:
 $smarty->assign('metatag_robots', 'NOINDEX, NOFOLLOW');
 ask_ticket('searchresults');
+$smarty->assign('pageExists',$tikilib->page_exists($words));
 $smarty->assign('searchNoResults', !isset($_REQUEST['words'])); // false is default
 $smarty->assign('mid', 'tiki-searchresults.tpl');
 $smarty->display("tiki.tpl");
