--- tiki-view_blog_post.php.orig	2010-06-07 19:26:45.000000000 -0300
+++ tiki-view_blog_post.php	2010-06-07 19:28:32.000000000 -0300
@@ -130,8 +130,7 @@
 if ($prefs['feature_freetags'] == 'y') {
 	// Get Tags
 	include_once ('lib/freetag/freetaglib.php');
-	$tags = $freetaglib->get_tags_on_object($postId, "blog post");
-	$smarty->assign('tags', $tags);
+	$post_info['freetags'] = $freetaglib->get_tags_on_object($postId, "blog post");
 }
 ask_ticket('view-blog-post');
 // Display the template
