--- lib/tikilib.php 2011-04-04 11:20:00.078371002 -0300
+++ lib/tikilib.php	2011-04-04 11:20:08.728370998 -0300
@@ -6595,10 +6595,13 @@
 								$listate = substr($line, $listlevel, 1);
 								if (($listate == '+' || $listate == '-') && !($litype == '*' && !strstr(current($listbeg), '</ul>') || $litype == '#' && !strstr(current($listbeg), '</ol>'))) {
 									$thisid = 'id' . microtime() * 1000000;
-									if ( !$options['ck_editor'] ) {
-										$data .= '<br /><a id="flipper' . $thisid . '" class="link" href="javascript:flipWithSign(\'' . $thisid . '\')">[' . ($listate == '-' ? '+' : '-') . ']</a>';
+									if (preg_match("|tiki-print.php|",$_SERVER['REQUEST_URI'])) {
+										// $data .= '<br /><img class="flipper" id="flipper' . $thisid . '" class="pointer" onclick="flip(\'' . $thisid . '\');toggleImage(this,\'iArrowGreyRight.png\')" src="styles/estudiolivre/iArrowGreyDown.png">';
+										$listyle = ' id="' . $thisid . '"';
+									} else {
+										$data .= '<br /><img class="flipper" id="flipper' . $thisid . '" class="pointer" onclick="flip(\'' . $thisid . '\');toggleImage(this,\'iArrowGrey' . ($listate == '-' ? 'Down' : 'Right') . '.png\')" src="styles/estudiolivre/iArrowGrey' . ($listate == '-' ? 'Right' : 'Down') . '.png">';
+										$listyle = ' id="' . $thisid . '" style="display:' . ($listate == '+' ? 'block' : 'none') . ';"';
 									}
-									$listyle = ' id="' . $thisid . '" style="display:' . ($listate == '+' || $options['ck_editor'] ? 'block' : 'none') . ';"';
 									$addremove = 1;
 								}
 							}
@@ -6612,9 +6615,10 @@
 						$listate = substr($line, $listlevel, 1);
 						if (($listate == '+' || $listate == '-')) {
 							$thisid = 'id' . microtime() * 1000000;
-							if ( !$options['ck_editor'] ) {
-								$data .= '<br /><a id="flipper' . $thisid . '" class="link" href="javascript:flipWithSign(\'' . $thisid . '\')">[' . ($listate == '-' ? '+' : '-') . ']</a>';
+							if (preg_match("|tiki-print.php|",$_SERVER['REQUEST_URI'])) {
+								$listate = '-';
 							}
+							$data .= '<br /><img class="flipper" id="flipper' . $thisid . '" class="pointer" onclick="flip(\'' . $thisid . '\');toggleImage(this,\'iArrowGrey' . ($listate == '-' ? 'Down' : 'Right') . '.png\')" src="styles/estudiolivre/iArrowGrey' . ($listate == '-' ? 'Right' : 'Down') . '.png">';
 							$listyle = ' id="' . $thisid . '" style="display:' . ($listate == '+' || $options['ck_editor'] ? 'block' : 'none') . ';"';
 							$addremove = 1;
 						}
@@ -6729,6 +6733,7 @@
 						$anchor = '';
 						$aclose = '';
 						$aclose2 = '';
+						$aopen = '';
 						$addremove = $show_title_level[$hdrlevel] ? 1 : 0; // If needed, also remove '#' sign from title beginning
 
 						// May be special signs present after '!'s?
@@ -6736,8 +6741,11 @@
 						if (($divstate == '+' || $divstate == '-') && !$options['ck_editor']) {
 							// OK. Must insert flipper after HEADER, and then open new div...
 							$thisid = 'id' . preg_replace('/[^a-zA-z0-9]/', '',urlencode($options['page'])) .$nb_hdrs;
-							$aclose = '<a id="flipper' . $thisid . '" class="link" href="javascript:flipWithSign(\'' . $thisid . '\')">[' . ($divstate == '-' ? '+' : '-') . ']</a>';
-							$aclose2 = '<div id="' . $thisid . '" class="showhide_heading" style="display:' . ($divstate == '+' ? 'block' : 'none') . ';">';
+							if(preg_match("|tiki-print.php|",$_SERVER['REQUEST_URI'])) {
+								$divstate = '+';
+							}
+							$aopen = '<span class="hiddenPointer" onclick="flip(\'' . $thisid . '\');toggleImage(document.getElementById(\'flipper' . $thisid . '\'),\'iArrowGrey' . ($divstate == '-' ? 'Down' : 'Right') . '.png\')"> <img class="flipper pointer" id="flipper' . $thisid . '" src="styles/estudiolivre/iArrowGrey' . ($divstate == '-' ? 'Right' : 'Down') . '.png">';
+							$aclose2 = '<div id="' . $thisid . '" style="display:' . ($divstate == '+' ? 'block' : 'none') . ';"> </span>';
 							global $headerlib;
 							$headerlib->add_jq_onready( "setheadingstate('$thisid');" );
 							array_unshift($divdepth, $hdrlevel);
@@ -6803,7 +6811,7 @@
 							++$headerInc;
 
 						if ( $prefs['feature_wiki_show_hide_before'] == 'y' ) {
-							$line = $button.'<h'.($hdrlevel+$headerInc).' class="showhide_heading" id="'.$thisid.'">'.$aclose.' '.$title_text.'</h'.($hdrlevel+$headerInc).'>'.$aclose2;
+							$line = $button.'<h'.($hdrlevel+$headerInc).' class="showhide_heading" id="'.$thisid.'">'.$aopen.' '.$title_text.'</h'.($hdrlevel+$headerInc).'>'.$aclose.$aclose2;
 						} else {
 							$line = $button.'<h'.($hdrlevel+$headerInc).' class="showhide_heading" id="'.$thisid.'">'.$title_text.'</h'.($hdrlevel+$headerInc).'>'.$aclose.$aclose2;
 						}
