--- lib/trackers/trackerlib.php	2006-10-18 14:59:37.000000000 -0300
+++ lib/trackers/trackerlib.php	2006-11-30 18:37:41.000000000 -0200
@@ -470,7 +470,7 @@
 		} else {
 
 			$query = "select tti.*, ttif.`value` from `tiki_tracker_items` tti, `tiki_tracker_item_fields` ttif $mid and tti.`itemId`=ttif.`itemId` group by tti.`itemId` order by ".$this->convert_sortmode($sort_mode);
-			$query_cant = "select count(distinct ttif.`itemId`) from `tiki_tracker_items` tti, `tiki_tracker_item_fields` ttif $mid and tti.`itemId`=ttif.`itemId` group by tti.`itemId`";
+			$query_cant = "select count(distinct ttif.`itemId`) from `tiki_tracker_items` tti, `tiki_tracker_item_fields` ttif $mid and tti.`itemId`=ttif.`itemId`";
 		}
 		if ($dbg) {
 			header('Content-type: text/plain');
