--- lib/userslib.php.orig	2010-05-03 17:11:53.000000000 -0300
+++ lib/userslib.php	2010-06-07 20:45:39.000000000 -0300
@@ -2977,6 +2977,21 @@
 		return $result;
 	}
 
+	function set_user_field($name, $value) {
+
+            global $user;
+
+       	    $aUserPrefs = array('realName','site','local', 'isPublic', 'licencaPadrao', 'allowMsgs');
+            if (in_array($name, $aUserPrefs)) {
+               $result = $this->set_user_preference($user, $name, $value);
+            } else {
+               $query = "UPDATE `users_users` set `$name`='$value' where `login`='$user'";
+               $result = $this->query($query);
+       	    }
+
+            return $result;
+    	}
+
 	function count_users($group) {
 		static $rv = array();
 
